site-src/v1alpha2/index.md

```diff
@@ -1,5 +1,7 @@
 ## What is the Gateway API?
 
+!!! danger "This version of the API has not been released yet."
```

Comment:
Had to duplicate the main page to distinguish between v1alpha1 and v1alpha2. At this point the only difference is the warning on the v1alpha2 page.